monitoring_service/server.py

```diff
@@ -67,6 +71,7 @@ def __init__(
                 address=monitor_contract_address
             )
         )
+        self.open_channels = set()
```

Comment:
As far as I can see this is not saved to the database, will this cause problems after a restart?

Reply:
If we don't start processing from the beginning of the chain, it will. We don't have any restart tests and I think we can fail in other places, too. Let's talk about this, before I fix this.

Reply:
I will take care of this as part of #37.